keras/src/ops/nn.py

```diff
@@ -76,6 +82,44 @@ def relu6(x):
         return Relu6().symbolic_call(x)
     return backend.nn.relu6(x)
 
+@keras_export(["keras.ops.flash_attention", "keras.ops.nn.flash_attention"])
```

Comment:
Hello there,

That's a very wonderful addition. I've a bit of doubt there:

I suppose `ops.flash_attention` is an *operation*, while `nn.flash_attention` is just a neural network layer.
The basic difference between these two is that - an *operation* **may not** have any trainable parameter with it, while a neural network layer **should** have trainable parameters.

Am I right till now?

If yes, please provide separate examples of each one of them in the docs!

Best Regards,
Abhas Kumar Sinha